README.md

```diff
@@ -18,10 +18,6 @@ npm i @google/events
 
 ## Generate
 
-Generate libraries:
+Generate libraries from quicktype (currently manual).
 
-```sh
-./gen.sh
-```
-
-Observe the new files in `src/`.
+https://app.quicktype.io/
```

Comment:
I think a bit more documentation is needed on what to put into quicktype.io and the process to collect that information. Would expect just a few bullets for now.

Reply:
Added a bit more documentation on the process!